packages/components/src/AsyncComponent/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { ScalprumComponent } from '@scalprum/react-core';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import classNames from 'classnames';
+
+export interface AsyncComponentProps {
```

Comment:
These props should extend a structure I call `AnyObject`. If we don't use that we forbid any other props that may be required by the child component.

Any object usually looks like this
```TSX
interface AnyObject {
  [key: string]: any
}
```